CHANGELOG.md

```diff
@@ -3,7 +3,48 @@
 
 ## Version 0.42.0
 
-In Development.
+([Full Changelog](https://github.com/unitaryfund/mitiq/compare/v0.41.0...v0.42.0))
+
+### Highlights
+
+🚀 Many thanks to **first time contributors** @gluonhiggs, @JMuff22 and @Shivansh20128!
+
+💡 A new error-mitigation technique is on its way. Thanks to @Misty-W for the Probabilistic Error Amplification RFC!
+
+🎏 Modular PEC functions are now available, courtesy of @natestemen!
```

Comment:
TIL 🎏!

Can we also add something like this:
````md
These functions allow a user to generate the intermediary sampled circuits and combine the results in a a two step process. E.g.

```py
from mitiq import pec

circuits = pec.intermediary_sampled_circuits(circuit, representations)

sampled_circuit_results = ...  # execute the circuits on a simulator/hardware/toothbrush/etc

pec_estimate = pec.combine_results(sampled_circuit_results, ...)
```
````

Reply:
> TIL 🎏!

Yeah! I got it from https://getemoji.com/#objects